config/webpack.config.js

```diff
@@ -244,10 +252,12 @@ const pluginConfig = function(plugin) {
       'webpack',
       pluginName
     );
+    var chunkFilename = '[name].js'
   }
   config.output = {
     path: outputPath,
     publicPath: '/webpack/' + pluginName + '/',
+    filename: chunkFilename,
```

Comment:
I am almost sure this will break plugins, but I've not tested it yet

Reply:
amazingly, it does not!

old foreman and old foreman_puppet:
![Screenshot from 2024-08-29 14-43-13](https://github.com/user-attachments/assets/fa34d1ed-6594-4f3e-bd2a-8a0896a6bec9)

new foreman (with hashes) and old foreman_puppet:
![Screenshot from 2024-08-29 14-42-44](https://github.com/user-attachments/assets/d4d713c3-1e21-4b51-99f1-dbedaa6bc04f)

new foreman (with hashes) and new foreman_puppet (with hashes):
![Screenshot from 2024-08-29 16-05-09](https://github.com/user-attachments/assets/28e9b117-3fb3-4d1d-82c0-c1dd79950caa)